From 4dac1a9a1d107342428015bd953267478f798ce7 Mon Sep 17 00:00:00 2001
From: Alejandro Enedino Hernandez Samaniego <alhe@linux.microsoft.com>
Date: Thu, 18 Feb 2021 16:21:33 -0700
Subject: [PATCH] debug uninative

Signed-off-by: Alejandro Enedino Hernandez Samaniego <alhe@linux.microsoft.com>
---
 meta/classes/uninative.bbclass | 6 ++++++
 1 file changed, 6 insertions(+)

diff --git a/meta/classes/uninative.bbclass b/meta/classes/uninative.bbclass
index 1e19917a97..766e0315db 100644
--- a/meta/classes/uninative.bbclass
+++ b/meta/classes/uninative.bbclass
@@ -58,12 +58,18 @@ python uninative_event_fetchloader() {
             # the current fetcher url handling
             premirrors = bb.fetch2.mirror_from_string(localdata.getVar("PREMIRRORS"))
             for line in premirrors:
+                bb.warn(str(line))
                 try:
                     (find, replace) = line
+                    bb.warn('find:%s' % find)
+                    bb.warn('repla:%s' % replace)
                 except ValueError:
                     continue
                 if find.startswith("http"):
+                    bb.warn('found')
+                    bb.warn(str(localdata.getVar('PREMIRRORS')))
                     localdata.appendVar("PREMIRRORS", " ${UNINATIVE_URL}${UNINATIVE_TARBALL} %s/uninative/%s/${UNINATIVE_TARBALL}" % (replace, chksum))
+                    bb.warn(str(localdata.getVar('PREMIRRORS')))
 
             srcuri = d.expand("${UNINATIVE_URL}${UNINATIVE_TARBALL};sha256sum=%s" % chksum)
             bb.note("Fetching uninative binary shim %s (will check PREMIRRORS first)" % srcuri)
-- 
2.25.1

